blocks/nav/__list/_type/nav__list_show.css

```diff
@@ -0,0 +1,4 @@
+.nav__list_show {
+  min-height: 100vh;
+  transition: all .3s linear;
```

Comment:
Usually when the burger menu is opened, users don't have the ability to scroll the page and click on the elements under the menu. It'd be really useful to add z-index and 'overflow-hidden' for root in this position.